postback.md

```diff
@@ -134,3 +134,25 @@ Subsequent postbacks are deferred until the first completes.
 Once the first postback request completes the defered requests will be processed sequentially.
 
 Since postback requests can ultimately fail, it's best to accept and store data on your end (with an HTTP 200 response to Signhost) for later processing to avoid data loss.
+
+### Postback attributes for iDIN
```

Comment:
This is not correct, these aren't Postback attributes perse but available once iDin verification is done. They will also be return on a GET request. This should be part of the iDIN documentation at https://evidos.github.io/endpoints/##/definitions/iDIN